tools/cli/commands/changelog.js

```diff
@@ -722,15 +730,21 @@ async function promptVersion( argv ) {
  *
  * @param {object} argv - the arguments passed.
  * @param {Array} needChangelog - projects that need changelog.
+ * @param {object} types - the changelog types for the projects.
+ *
  * @returns {argv}.
  */
-async function promptChangelog( argv, needChangelog ) {
+async function promptChangelog( argv, needChangelog, types ) {
 	const gitBranch = child_process
 		.spawnSync( 'git', [ 'branch', '--show-current' ] )
 		.stdout.toString()
 		.trim()
 		.replace( /\//g, '-' );
-	console.log( gitBranch );
+	const maxLength = Math.max( ...Object.keys( types ).map( key => key.length ) );
```

Comment:
`Array.reduce` doesn't get used often enough. 😀
```suggestion
	const maxLength = Object.keys( types ).reduce( ( a, v ) => v.length > a ? v.length : a, 0 );
```

Reply:
Oooh very fancy.